### PR TITLE
20 class

### PR DESCRIPTION
**Pull request by igneel73**
added 20 classes, and made minor css fixes

### CODE REVIEW

File: app/static/js/spcconvert.js [hunk 1]
```diff
@@ -446,25 +446,40 @@ function setGtruth() {
...
+        $('#TargetImg-annot').height(0.75*$('#TargetImg-annot').height());
+        $('#TargetImg-annot').width(0.75*$('#TargetImg-annot').height()/aspectRatio);
     }
 
+    $('#TargetImg-annot').attr("src", "static/" +incorrectArr[img_i]);
+    $('#ImageDetailTitle-annot').html(incorrectArr[img_i]);
+    $('#ImageName-annot').html("<span class='info-label'>Image ID</span>" + "<span>" + img.image_id + "</span>");
+    $('#Timestamp-annot').html("<span class='info-label'>Collection Datetime</span>" + "<span>" + img.image_timestamp + "</span>");
+    // Info
+    var classes = {};
```
Comment (ktl014) [inline app/static/js/spcconvert.js:475]: I'm assuming these are labels for when we label right? Are they still being used (i.e. how come the other 20 classes aren't listed)?

File: app/templates/index.html [hunk 2]
```diff
@@ -135,31 +135,31 @@ <h4>Prediction Chart</h4>
...
-                        <option value="12" />
-                        <option value="13" />
-                        <option value="14" />
-                        <option value="15" />
-                        <option value="16" />
-                        <option value="17" />
-                        <option value="18" />
-                        <option value="19" />
-                        <option value="20" />
+                        <option value="0" >Prorocentrum micans</option>
```
Comment (ktl014) [inline app/templates/index.html:141]: Did you verify with Sarah if these are the correct labels to enter?

**Review by ktl014 — APPROVED** [review]

**Comment by ktl014** [comment]
Give a reply, then you can merge it.

File: app/templates/index.html (hunk 2 above)
Comment (igneel73) [reply app/templates/index.html:141]: Yes, I took them straight from the document Melissa sent.

File: app/static/js/spcconvert.js (hunk 1 above)
Comment (igneel73) [reply app/static/js/spcconvert.js:475]: These are just to show the prediction probabilities given by the classifier in the sidebar. More classes will be added as we expand the classifier.

**Pull request merged**